src/components/form-draft/testing.vue

```diff
@@ -21,6 +21,25 @@ except according to the terms contained in the LICENSE file.
             </enketo-fill>
           </template>
           <template #body>
+            <div v-if="form && form.entityRelated" class="panel-dialog">
```

Comment:
`form` will now be a resource object and will never be `null`. You can check whether the resource has data using `form.dataExists`.